c10/core/TensorImpl.h

```diff
@@ -2215,6 +2225,11 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
 
   Storage storage_;
 
+ public:
+  void set_custom_strides_policy() {
+    set_sizes_strides_policy(SizesStridesPolicy::CustomStrides);
```

Comment:
`set_sizes_strides_policy` is public, might as well just use it directly (or if it's not public, let's make it public)

Reply:
Sounds good - it's protected right now, so I'll just make it public!